index.js

```diff
@@ -132,7 +132,9 @@ const _onBeforeInput = shortcutsOfWindow => (e, input) => {
 	for (const {eventStamp, callback} of shortcutsOfWindow) {
 		if (equals(eventStamp, event)) {
 			debug(`eventStamp: ${eventStamp} match`);
-			callback();
+			if (callback()) {
```

Comment:
Curious why not pass `e` to the callback and let the downstream code preventDefault? That feels more "normal" like working with typical event handler callbacks.

Reply:
I created this from our use case and we always return true to prevent Electron from handling events (c&p from our fork). I also prefer the cleaner control flow and don't think it's necessary to pass the whole event, but on the other side, others may require the event.